ssz-rs/src/list.rs

```diff
@@ -294,7 +297,7 @@ pub struct IterMut<'a, T, const N: usize>
 where
     T: SimpleSerialize,
 {
-    inner: Enumerate<slice::IterMut<'a, T>>,
+    inner: Enumerate<crate::lib::IterMut<'a, T>>,
```

Comment:
if we did `use crate::lib::*;` up above, `IterMut` should be in scope, right?

Reply:
It looks like `IterMut` conflicts with the struct definition `pub struct IterMut<'a, T, const N: usize>`, so I have had to specify that it is the `IterMut` from the lib.